src/mapeo-project.js

```diff
@@ -139,11 +144,37 @@ export class MapeoProject extends TypedEmitter {
     this.#isArchiveDevice = isArchiveDevice
 
     ///////// 1. Setup database
+
     this.#sqlite = new Database(dbPath)
     const db = drizzle(this.#sqlite)
-    migrate(db, { migrationsFolder: projectMigrationsFolder })
+    const migrationsBefore = tableCountIfExists(db, DRIZZLE_MIGRATIONS_TABLE)
+    migrate(db, {
+      migrationsFolder: projectMigrationsFolder,
+      migrationsTable: DRIZZLE_MIGRATIONS_TABLE,
+    })
+    const migrationsAfter = tableCountIfExists(db, DRIZZLE_MIGRATIONS_TABLE)
+    const reindex = migrationsBefore > 0 && migrationsAfter !== migrationsBefore
```

Comment:
Maybe wrap this in our own migrate function for clarity? e.g. 
```
const didMigrate = migrate(db, { migrationsFolder: projectMigrationsFolder })
```

Reply:
Done in ae6e0c6d4d97b160f5336b79a578f80bc1aa7085.